review-cask-pr/review.rb

```diff
@@ -177,8 +177,11 @@ def review_diff(pr, author_is_member: false)
 
       puts message
 
-      puts "::set-output name=event::#{event}"
-      puts "::set-output name=message::#{GitHub::Actions.escape(message)}"
+      github_output = ENV.fetch("GITHUB_OUTPUT") { raise "GITHUB_OUTPUT is not defined" }
+      File.open(github_output, "a") do |f|
+        f.puts("event=#{event}")
+        f.puts("message=#{GitHub::Actions.escape(message)}")
```

Comment:
Need to check if escaping work the same way (I can check this - just noting here for now).

Reply:
Answer to this is yes: this needs changing. Not sure if it's best to do here or on the brew side. This might be the only place that uses it for GITHUB_OUTPUT?